app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,44 @@
+  <h4>Lunch</h4>
+   <p>Lunch is an opportunity for teachers to have a well-earned break and go to the staff room to talk about how the morning has been. Many teachers also use this as an opportunity to get ahead on their marking. By tackling some books during this time, it can reduce the amount of marking needed to be done in the afternoon.</p>
+   <%= render 'content/shared/quotes/quote_secondary_mike' %>
+ <h3>Afternoon</h3>
+  <h4>2:15pm afternoon lessons</h4>
+   <p>After lunch, there's often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.</p>
+  <h4>3:15pm pupils leave school</h4>
+   <p>When pupils leave school the work continues. Teachers generally spend this time marking work and planning upcoming lessons.</p>
+   <p>Some teachers will also run an afterschool club such as a football team, drama sessions or computer club.</p>
+   <%= render 'content/shared/quotes/quote_ict_gavin' %>
+  <h4>5:00pm leave school</h4>
+   <p>Depending on how the teacher likes to split their work, they will leave school between 4:30pm and 6:00pm. Some teachers will take home books to mark or carry on planning once they're at home.</p>
+  <h3>Planning preparation and assessment time (PPA)</h4>
+   <p>Teachers are entitled to 10% of their time without lessons, so they can plan work. Depending on the school, this could mean that you have a couple of free periods next to each other or they could be spread across the week.</p>
+
+<h2 class="heading--box-blue">Support for teachers</h2>
+ <h3>Managing behaviour</h3>
+  <p>Teaching is not just about delivering content. It also involves helping pupils understand how to behave appropriately in school. Pupils will test limits, become disruptive, or fail to follow instructions at times, which can be daunting.</p>
+  <p>However, training courses will provide you with strategies to boost your confidence and make you feel prepared. Schools will also have behaviour policies in place which you can follow and colleagues will help support you.</p>
+  <%= render 'content/shared/quotes/quote_behaviour_jane' %>
+
+<h3>Workload</h3>
+ <p>Teaching around 5 lessons a day to classes of 30 pupils can create a lot of admin. Teachers spend a lot of their time outside of lessons marking work and planning lessons.</p>
+ <p> When you are training, you will work with a teacher who will help you plan and may share some of their own planning with you. Over time, with the help of other colleagues, you will get a better feel for what makes a good lesson and build a bank of resources you can reuse cutting down on your workload.</p>
+ <%= render 'content/shared/quotes/quote_food_helen' %>
+<p>To find out more about what it's like to be a teacher, you can visit schools to <a href="/train-to-be-a-teacher/get-school-experience">get school experience to see if it is the right career for you</a>.</p>
```

Comment:
I'm not sure if signposting to GSE is the right fit, as it's only available to people with a degree. We could signpost to the advisers using the new adviser promo component?